src/ShallowWrapper.js

```diff
@@ -520,6 +521,29 @@ export default class ShallowWrapper {
   }
 
   /**
+   * Return a JSON representation of the node.
+   *
+   * NOTE: can only be called on a wrapper of a single node.
+   *
+   * @returns {Object}
+   */
+  json() {
+    return this.single('json', () => {
+      if (this.type() == null) {
+        return this.node;
+      }
+
+      const childrens = compact(this.children().map(c => c.json()));
+      return {
+        type: this.name(),
+        props: omit(this.props(), 'children'),
+        children: (childrens.length) ? childrens : null,
+        $$typeof: Symbol.for('react.test.json'),
+      };
```

Comment:
If it were, it would need to be called `toJSON` anyways. I agree that the name `json` isn't appropriate.

The catch is that enzyme, and its tests, need to run in many browsers and all supported nodes, if we want to be able to test how our React components actually work in these environments. Relying on Symbols hurts that goal.